### PR TITLE
docs(remediate): update remediate docs

### PR DESCRIPTION
**Pull request by cxMiguelSilva**
**Proposed Changes**
- update remediate docs

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

**Review by nunoocx — APPROVED** [review]
LGTM

File: docs/kics_auto_remediation.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ Note that this feature will be only available for Terraform, for now.
    </p>
 
 
-2. If your JSON report has any result with remediation, you will need to run the new KICS command: fix. 
+2. If your JSON report has any result with remediation, you will need to run the new KICS command: remediate. 
 
-   If you want KICS to remediate all the reported issues, you can run ```docker run -v /home/cosmicgirl/:/path/ kics fix --results /path/results/results.json -v```.
```
Comment (gabriel-cx) [inline docs/kics_auto_remediation.md:28]: Here we can give a new line, so the command is shown in a new line.

File: docs/kics_auto_remediation.md [hunk 2]
```diff
@@ -23,8 +23,8 @@ Note that this feature will be only available for Terraform, for now.
...
+   If you want KICS to remediate all the reported issues, you can run ```docker run -v /home/cosmicgirl/:/path/ kics remediate --results /path/results/results.json -v```.
 
-   If you want to specify which remediation KICS should fix, you can use the flag `--include-ids`. In this flag, you should point the `similarity_id` of the result. For example: ```docker run -v /home/cosmicgirl/:/path/ kics fix --results /path/results/results.json --include-ids "f282fa13cf5e4ffd4bbb0ee2059f8d0240edcd2ca54b3bb71633145d961de5ce" -v```
```
Comment (gabriel-cx) [inline docs/kics_auto_remediation.md:30]: Here we can give a new line, so the command is shown in a new line.

**Review by gabriel-cx — CHANGES_REQUESTED** [review]

**Review by gabriel-cx — APPROVED** [review]
LGTM

**Pull request merged**